jobs/config.json

```diff
@@ -7604,6 +7604,29 @@
       "sig-cluster-lifecycle"
     ]
   },
+  "ci-kubernetes-e2e-kubeadm-gce-upgrade-stable-master": {
```

Comment:
call it `periodic-` as it is a periodic job

Reply:
@luxas all the other kubeadm jobs are with `ci-`, I think `ci-` is fine and we should simply avoid put `ci-` for postsubmit jobs

Reply:
ok